docs/README.md

```diff
@@ -12,6 +12,7 @@ In this folder are the following docs:
 * [API Specifications](./api-specifications.md)
 * [Configuration](./configuration.md)
 * [Monitoring](./monitoring.md)
+* [Database Maintenance](./database-maintenance.md)
 * [Database migrations](./database_migrations.md)
```

Comment:
Can we make that uppercase to make it consistent?

Reply:
Changed.